docs/antora/modules/ROOT/pages/setup/docker.adoc

```diff
@@ -16,54 +17,67 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_docker_linux_post_install: https://docs.docker.com/install/linux/linux-postinstall/
 :url_xcode: https://developer.apple.com/xcode/features/
 
+:lisk-docs: ROOT::
 :url_index_usage: index.adoc#usage
 :url_setup: setup/index.adoc
 :url_core_setup_npm: master@lisk-core::setup/npm.adoc
 :url_config: configuration/docker.adoc
 :url_management: management/docker.adoc
-:url_references_config: reference/configuration.adoc
+:url_references_config: configuration/index.adoc
+:url_build_blockchain:  {lisk-docs}build-blockchain/create-blockchain-app.adoc
 
 :url_core_setup_binary: master@lisk-core::setup/binary.adoc
 
 How to set up Lisk Service and connect it to a local Lisk Core node.
 
-== Setting up Lisk Core
+[NOTE]
+====
+*Pre-requisite*
 
-Lisk Service is installed in addition to a Lisk blockchain application such as Lisk Core, in order to provide enriched network data for third-party services.
+Lisk Service is a web application middleware that allows interaction with various blockchain networks based on the Lisk protocol.
+It is recommended to set up a blockchain node first, before setting up Lisk Service.
 
-The first step is therefore to install Lisk Core.
+* To set up a Lisk-Core node, see any of our setup guides, such as xref:{url_core_setup_npm}[].
+* Alternatively, you can setup up a sidechain node to connect to Lisk Service.
```

Comment:
Could you also please add a note to emphasize that setting the `system.keepEventsForHeights: -1` in the config is very important when setting up the node?

```suggestion
* To set up a Lisk-Core node, see any of our setup guides, such as xref:{url_core_setup_npm}[].
* Alternatively, you can setup up a sidechain node to connect to Lisk Service.
* Please make sure to set `system.keepEventsForHeights: -1` in the node config before synchronizing the node with the network.
```